src/adapters/postgres/user-adapter.ts

```
@@ -1706,4 +1718,185 @@ export class PostgresUserService implements IServicelocator {
+      const { mobile, otp, hash } = body;
+      if (!mobile || !otp || !hash) {
+        return APIResponse.error(
+          response,
+          apiId,
+          API_RESPONSES.BAD_REQUEST,
+          API_RESPONSES.OTP_VALIDED_REQUIRED_KEY,
+          HttpStatus.BAD_REQUEST
+        );
+      }
+      const [hashValue, expires] = hash.split('.');
+      if (!hashValue || !expires || isNaN(parseInt(expires))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)